src/main/scala/workflow/graph/LabelEstimator.scala

```diff
@@ -0,0 +1,108 @@
+package workflow.graph
+
+import org.apache.spark.rdd.RDD
+
+/**
+ * A LabelEstimator has a `fitRDDs` method which takes input data and input labels,
```

Comment:
Oh, I see now - you ask them to implement the type-safe thing. I guess this comes back to my naming thing above.